src/components/shell/NavigationMenu.tsx

```diff
@@ -0,0 +1,76 @@
+import { AccountCircle as AccountCircleIcon } from '@mui/icons-material'
+import { Box, Divider, IconButton, Typography } from '@mui/material'
+import Menu from '@mui/material/Menu'
+import MenuItem from '@mui/material/MenuItem'
+import * as React from 'react'
+
+interface NavigationMenuProp {}
+
+const NavigationMenu: React.FC<NavigationMenuProp> = () => {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null)
+  const open = Boolean(anchorEl)
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget)
+  }
+  const handleClose = () => {
+    setAnchorEl(null)
+  }
+
+  return (
+    <>
+      <IconButton
+        aria-controls={open ? 'basic-menu' : undefined}
+        aria-haspopup="true"
+        aria-expanded={open ? 'true' : undefined}
+        onClick={handleClick}
+        size="large"
+        edge="end"
+      >
+        <AccountCircleIcon />
+      </IconButton>
+      <Menu
+        id="basic-menu"
+        anchorEl={anchorEl}
+        open={open}
+        onClose={handleClose}
+        MenuListProps={{
+          'aria-labelledby': 'basic-button'
+        }}
+      >
+        <MenuItem onClick={handleClose}>
+          <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+            <Typography
+              fontWeight={400}
+              fontSize={'16px'}
+              lineHeight="150%"
+              letterSpacing={'0.15px'}
+            >
+              John Schmidt
+            </Typography>
+            <Typography
+              fontWeight={400}
+              fontSize={'14px'}
+              lineHeight="143%"
+              letterSpacing={'0.17px'}
+            >
+              johnshmidt@match.com
```

Comment:
I know we are not there yet, How about passing a variable already to cater for when the API sends the user info? Or even use the mock backend for it? What do you think?

Reply:
@IlyesBenAmara You can mock a API endpoint "GET /users/me" that would return a user object : {id, username, email}

Reply:
BWT there's a useAuth() hook that is supposed to return the user object. So you need to get the current user with the AuthContextProvider

Reply:
Understood, working on it :technologist: